r/1. Getting Started/Getting Started.Rmd

```diff
@@ -0,0 +1,172 @@
+---
+title: "Getting started with openEO"
+---
+
+openEO is an open-source initiative that simplifies accessing and processing Earth Observation (EO) data.
+
+Traditional methods involve complex steps like data discovery, download, and pre-processing, which can be time-consuming and challenging, especially when dealing with multiple datasets. openEO standardises this process, providing a unified interface for accessing and processing diverse EO datasets using familiar programming languages like Python, etc. It leverages the concept of datacubes, which streamline the representation and manipulation of EO data, making spatiotemporal analysis more intuitive and efficient.
+
+openEO is used in several applications across a range of EO scenarios, ranging from simple to complex workflows. However, this notebook aims to guide beginners in starting with openEO using the R client.
+
+This notebook is based on existing openEO examples. For more detailed explanations, we recommend checking out the sample notebooks. For a deeper theoretical understanding, you might also find the Eo-college course ["Cubes&Cloud"](https://eo-college.org/courses/cubes-and-clouds/) helpful, as it offers step-by-step guidance and theoretical insights. While the course focuses on the [openEO Python client](https://open-eo.github.io/openeo-python-client/index.html), the concepts still apply when using the R client.
+
+Here, our focus is to help users become acquainted with the general openEO workflow using R. Additionally, we recommend visiting the official [openEO R client](https://open-eo.github.io/openeo-r-client/) documentation for more detailed information on the available functions and their usage.
+
+
+## Installation
+
+Before installing the openEO R-client module, ensure you have atleast version 3.6 of R. Older versions might work but haven't been tested.
+
+You can install stable releases from CRAN:
+
+```{r}
+install.packages("openeo")
+```
+
+You can find additional information on openEO installation in [this page](https://open-eo.github.io/openeo-r-client/#installation).
+
+```{r}
+library(openeo)
+
+```
+
+
+## Connect and authenticate
+
+Next, let's set up a connection to an openEO backend using its connection URL. You can find these URLs for different backends on the [openEO hub](https://hub.openeo.org/). For this notebook, we'll use the Copernicus Data Space Ecosystem, a cloud platform supported by the European Commission, ESA, and Copernicus. Make sure you have an [account](https://identity.dataspace.copernicus.eu/auth/realms/CDSE/protocol/openid-connect/auth?client_id=cdse-public&response_type=code&scope=openid&redirect_uri=https%3A//dataspace.copernicus.eu/account/confirmed/1) to access and process data using openEO.
+
+When using other backends, you can register using your EduGAIN and social login as suggested [here](https://docs.openeo.cloud/join/free_trial.html)
+
+```{r}
+connection = connect(host = "https://openeo.dataspace.copernicus.eu")
```

Comment:
Apparently the way the R client works, it's not necessary to specify your connection in `create_job` once your are connected. So maybe it's more intuitive to not store the connection in a variable?
Instead just say
```R
connect(host = "https://openeo.dataspace.copernicus.eu")
```

Reply:
Indeed. Updated 😅